corehq/apps/case_importer/tracking/views.py

```diff
@@ -121,8 +121,9 @@ def case_upload_form_ids(request, domain, upload_id):
 
     ids_stream = ('{}\n'.format(form_id)
                   for form_id in get_form_ids_for_case_upload(case_upload))
-
-    return StreamingHttpResponse(ids_stream, content_type='text/plain')
+    response = StreamingHttpResponse(ids_stream, content_type='text/plain')
+    set_file_download(response, f"{domain}-case_upload-form_ids.txt")
```

Comment:
Does this change just set the filename?

Reply:
This change makes it show up as a "download" instead of just opening a page in the browser with a list of IDs.

Reply:
Cool, for some reason I thought this already did a download...I think the icon tricked me.